.github/workflows/macos_alpha_release.yml

```diff
@@ -0,0 +1,177 @@
+      AWS_SECRET_ACCESS_KEY_RELEASE_S3: ${{ secrets.AWS_SECRET_ACCESS_KEY_RELEASE_S3 }}
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+      MM_WEBHOOK_URL: ${{ secrets.MM_WEBHOOK_URL }}
+      SSH_PRIVATE_KEY_FASTLANE_MATCH: ${{ secrets.SSH_PRIVATE_KEY_FASTLANE_MATCH }}
+
+  appstore-release:
+    name: Prepare AppStore Release
+
+    needs: calculate-alpha-build-number
+
+    if: inputs.skip-appstore != 'true'
+
+    uses: ./.github/workflows/macos_build_appstore.yml
+    with:
+      destination: testflight_alpha
+      branch: ${{ inputs.branch }}
+      build-number-override: ${{ needs.calculate-alpha-build-number.outputs.build-number }}
+    secrets:
+      SSH_PRIVATE_KEY_FASTLANE_MATCH: ${{ secrets.SSH_PRIVATE_KEY_FASTLANE_MATCH }}
+      APPLE_API_KEY_BASE64: ${{ secrets.APPLE_API_KEY_BASE64 }}
+      APPLE_API_KEY_ID: ${{ secrets.APPLE_API_KEY_ID }}
+      APPLE_API_KEY_ISSUER: ${{ secrets.APPLE_API_KEY_ISSUER }}
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+      ASANA_ACCESS_TOKEN: ${{ secrets.ASANA_ACCESS_TOKEN }}
+      MM_WEBHOOK_URL: ${{ secrets.MM_WEBHOOK_URL }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  publish-alpha-dmg:
+    name: Publish Alpha DMG
+
+    needs: dmg-release
+
+    runs-on: macos-15
+
+    defaults:
+      run:
+        working-directory: .
+
+    steps:
+      - name: Download DMG URL artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dmg_url
+
+      - name: Set DMG URL variable
+        id: set-dmg-url
+        run: echo "DMG_URL=$(<${{ github.workspace }}/dmg_url.txt)" >> $GITHUB_OUTPUT
+
+      - name: Set up Sparkle tools
+        env:
+          SPARKLE_URL: https://github.com/sparkle-project/Sparkle/releases/download/${{ vars.SPARKLE_VERSION }}/Sparkle-${{ vars.SPARKLE_VERSION }}.tar.xz
+        run: |
+          curl -fLSs $SPARKLE_URL | tar xJ bin
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+          mkdir -p sparkle
+
+      - name: Fetch DMG and current appcast
+        id: fetch-dmg
+        working-directory: sparkle
+        env:
+          APPCAST_URL: ${{ vars.DMG_URL_ROOT }}alpha/appcast-alpha.xml
+          DMG_URL: ${{ steps.set-dmg-url.outputs.DMG_URL }}
+        run: |
+          curl -fLSs -O "${{ env.APPCAST_URL }}"
+          curl -fLSs -O "${{ env.DMG_URL }}"
+          echo "dmg-name=$(basename "${{ env.DMG_URL }}")" >> $GITHUB_OUTPUT
+          echo "dmg-path=$DMG_NAME" >> $GITHUB_OUTPUT
+
+      - name: Add release notes
+        id: add-release-notes
+        working-directory: sparkle
+        env:
+          DMG_NAME: ${{ steps.fetch-dmg.outputs.dmg-name }}
+        run: |
+          echo <<EOF > ${DMG_NAME/dmg/html}
+          Alpha release built from <a href="https://github.com/duckduckgo/apple-browsers/commits/main/?since=${{ github.sha }}">commit ${{ github.sha }}</a>.
```

Comment:
Release notes linking to the commit log starting with current sha. It’s something ;)

Reply:
I had to remove it because HTML tags aren’t supported in the release notes page. I will later add the link elsewhere in the app (“About" settings perhaps) to the commit history.